.azure/build.yaml

```
@@ -9,31 +9,35 @@ variables:
   - group: choreo-samples
 
 steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.x'
+      addToPath: true
+
+  - script: |
+      python -m pip install --upgrade pip
+      pip install pyyaml
+    displayName: 'Install dependencies'
+
+  - script: python .azure/process-metadata.py 
```

```suggestion
  - script: python .azure/process_metadata.py 
```

shall we use underscore for python files?